models/news.models.js

```diff
@@ -136,4 +136,19 @@ exports.updateArticleById = (reqBody, reqParams) => {
     }).catch((err)=>{
         return Promise.reject(err);
     });
+}
+
+exports.removeCommentById = (reqParams) => {
+    return db.query(`
+    DELETE FROM comments
+    WHERE comment_id = $1
+    RETURNING *`, [reqParams.comment_id])
+    .then((response)=>{
+        if (response.rows.length === 0){
+            return Promise.reject({msg: 'Not Found'})
+        }
+        return response.rows;
+    }).catch((err)=>{
```

Comment:
We should only have one .catch block at the end of our promise chain which in this case is in our controller, please remove.